dissect/target/plugins/os/windows/log/evtx.py

```diff
@@ -82,9 +82,11 @@ def evtx(self, log_file_glob: str = EVTX_GLOB, logs_dir: str | None = None) -> I
                 continue
 
             self.target.log.info("Processing event log file %s", entry)
-
-            for event in evtx.Evtx(entry_data):
-                yield self._build_record(event, entry)
+            try:
+                for event in evtx.Evtx(entry_data):
+                    yield self._build_record(event, entry)
```

Comment:
```suggestion
                    try:
                        yield self._build_record(event, entry)
                    except Exception as e:
                        self.target.log.warning("Unable to parse event log entry %s: %s", event, e)
                        self.target.log.debug("", exc_info=e)
```
So we don't skip the entire file if only one entry fails.